### PR TITLE
Update BUILDING

### PR DESCRIPTION
**Pull request by stephansnyt**
I'm trying to build from scratch, having never done any drone development before and I needed to do the "install drone" step before the last step worked.

### CODE REVIEW

File: BUILDING [hunk 1]
```diff
@@ -6,8 +6,17 @@
...
+
+4. Install binaries to $GOPATH/bin
 
     go install github.com/drone/drone/cmd/drone-agent
     go install github.com/drone/drone/cmd/drone-server
+
+5. Test
+
+   cd $GOPATH/src/github.com/drone/drone
+   go test -cover $(go list ./... | grep -v /vendor/)
```
Comment (bradrydzewski) [inline BUILDING]: This actually changes with Go 1.10. The `go test` command excludes the vendor directory automatically. So the syntax could be simplified to this:

```
go test github.com/drone/drone/...
```

File: BUILDING [hunk 2]
```diff
@@ -6,8 +6,17 @@
     go get -u golang.org/x/net/context/ctxhttp
     go get -u github.com/golang/protobuf/proto
     go get -u github.com/golang/protobuf/protoc-gen-go
+    
+3. Install drone
 
-3. Install binaries to $GOPATH/bin
+    go get -u github.com/drone/drone/...
```
[the remark this answers is not in the document]
Comment (tboerger) [reply BUILDING]: And if you add this step than it should use -d for just download the sources

[the remark this answers is not in the document]
Comment (stephansnyt) [reply BUILDING]: Maybe this is due to my lack of familiarity with the go toolchain, but I tend not to like cloning manually into my GOPATH and rely on `go get` to do everything for me.

I changed this to use `-d`

File: BUILDING (hunk 1 above)
Comment (stephansnyt) [reply BUILDING]: The current build for the project still use `1.9`, I pulled this line straight from `.drone.yml`

**Comment by stephansnyt** [comment]
Like I said above, I'm not that familiar with the golang toolchain. The goal of my changes were to produce a doc that anyone can cut and paste to get a working build (provided they can install git and go).

**Comment by tboerger** [comment]
As you have switched to a markdown file please also use markdown headlines instead of the big code block, otherwise LGTM.

**Comment by bradrydzewski** [comment]
Feels to complex. Why not just simplify and say this:

```
To build, test and install drone, please use the standard Go toolchain:

    go get github.com/drone/drone/...
    go test github.com/drone/drone/...
    go install github.com/drone/drone/...
```

**Comment by appleboy** [comment]
agree with @tboerger

**Pull request closed without merging**